lib/gears/timer.lua

```diff
@@ -104,9 +146,14 @@ local timer_instance_mt = {
 --- Create a new timer object.
 -- @tparam table args Arguments.
 -- @tparam number args.timeout Timeout in seconds (e.g. 1.5).
+-- @tparam[opt=false] boolean args.autostart Automatically start the timer.
+-- @tparam[opt=nil] function args.callback Callback function to connect to the
+--  "timeout" signal.
+-- @tparam[opt=false] boolean args.single_shot Run only once then stop.
```

Comment:
Qt call it `singleShot` and glib doesn't have it. My naming process is usually glib->qt->random. `once` isn't a bad name, but so is `single_shot`. If you *really* want me to change it, I will, but right now I left it as it is.